app/services/foreman_ansible/operating_system_parser.rb

```diff
@@ -81,6 +81,11 @@ def os_name
             facts[:ansible_distribution].tr(" \n\t", '')
       else
         distribution = facts[:ansible_lsb].try(:[], 'id') || facts[:ansible_distribution]
+        # RHEL 7 is marked as either RedHatEnterpriseServer or RedHatEnterpriseWorkstation, RHEL 8 is lsb id is RedHatEnterprise
+        # but we always consider it just RHEL on this level, workstation is differentiated below
+        if %w(RedHatEnterprise RedHatEnterpriseServer RedHatEnterpriseWorkstation).include?(distribution)
+          distribution = 'RedHat'
```

Comment:
actually we have `distribution = RedHat` and later on we pick the same fact and do something else (add `_Workstation`)
Can't we join those two if statements?